packer.py

```diff
@@ -48,24 +52,26 @@ def create_bins(bins: dict) -> list:
             bins[bin]['max_weight']))
     return bins_list
 
+
 def refresh_items(unfitted_items, packer):
```

Comment:
This name of function sounds strange. `refresh`....But we do not refresh, but `add` items to packer. So I suggest to rename it.

Reply:
This is right. Moreover, it uses the method `add_item()` from `main.py` module. The difference is that this `refresh...` function applies the 'add_item()` to the list of items one by one.
In this case - is this better to just rename the function and keep it, **or** to rewrite the code utilising `add_item()` method instead?

Reply:
We can always move the `refresh...()` functions to Packer class and make it a method `Packer.append([item] -> list)`. 
But I'm not sure if that is not just duplicating the `add...()` methods, which can be easily applied in the for-loop to do the same.